src/lib/response-triggers/results.ts

```diff
@@ -62,7 +62,7 @@ export default class Results extends Trigger {
 
 	async createResponse(data : ResultRow[]) : Promise<MessageObject> {
 		let today = new Date();
-		let todayString = today.getFullYear() + '-' + (today.getMonth() + 1) + '-' + today.getDate();
+		let todayString = today.getFullYear() + '&ndash;' + (today.getMonth() + 1) + '&ndash;' + today.getDate();
 
 		let filePath = `${__dirname}/../../../pegs-${todayString}`;
```

Comment:
Um, is it wise to change the todayString here if it's being used as the file path?

Reply:
No, especially since it shouldn't have used n dashes there anyway. ISO standard specifies hyphens.